src/components/transactions/Repay/CollateralRepayActions.tsx

```diff
@@ -44,35 +40,26 @@ export const CollateralRepayActions = ({
   maxSlippage,
   ...props
 }: RepayActionProps) => {
-  const { lendingPool } = useTxBuilderContext();
-  const { currentChainId: chainId, currentNetworkConfig } = useProtocolDataContext();
-  const { currentAccount } = useWeb3Context();
+  // do we need currentAccount in deps?
+  const currentAccount = useRootStore((state) => state.account);
```

Comment:
Other methods like `supply` are accessing `currentAccount` inside of `poolSlice` so this isn't needed here

Reply:
good catch, @satanworker can you please have a look at all the actions? I think most of them fetch & pass the account which should be handled inside the action.

Reply:
done